atom/browser/api/atom_api_app.cc

```diff
@@ -525,6 +525,7 @@ void App::OnLogin(LoginHandler* login_handler,
 void App::OnCreateWindow(const GURL& target_url,
                          const std::string& frame_name,
                          WindowOpenDisposition disposition,
+						 const std::vector<base::string16>& features,
```

Comment:
This indenting seems off by quite a bit, it should be aligned with the line above